README.md

```diff
@@ -27,6 +27,7 @@ Before working with the Jekyll code, you will need to install Jekyll:
     $ gem install jekyll
     $ gem install jekyll-redirect-from
     $ gem install html-proofer
+    $ gem install activesupport -v 4.2.7
```

Comment:
I don't believe I've needed this on two recent fresh installs on Mac, though I did need 'bundler', which was just added in pull/42. (Also I recall general frustration with 'gem' installation not seeming to do the right transitive things.)

I know @tgroh, @melap, and @eljefe6a have recently gone through this -- any thoughts from you folks?